src/nncf/quantization/algorithms/weight_compression/scale_estimation.py

```diff
@@ -211,26 +211,25 @@ def calculate_quantization_params(
         cur_config = deepcopy(config)
         cur_config.group_size = group_size
 
-        original_weight = fns.zeros_like(weight) + weight
```

Comment:
Values of original weight are not changed in-place below, so should be safe to reuse the same tensor.